content/docs/en/elements/components/image.md

```diff
@@ -1,19 +1,23 @@
 ---
 title: Image
-apiRef: https://docs.nativescript.org/api-reference/classes/_ui_image_.image
+apiRef: https://docs.nativescript.org/api-reference/classes/_ui_button_.button
 ---
 
-The Image component is used to display images.
+The Image widget shows the image from an ImageSource or from an URL.
 
 ---
 
 ```html
-<Image src="res://icon" stretch="none"/>
```

Comment:
I would keep this as a 3rd example since it is used fairly often

Reply:
Good idea!